src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -48,9 +48,7 @@
  * <pre>
  * CloudantClient client = new CloudantClient("mycloudantaccount","myusername","mypassword");
  * </pre>
- * <p/>

```

Comment:
I think you need to fix up the paragraph tags throughout this javadoc block as they don't balance even after your changes. If it's a set of steps the user needs to go through then maybe a list (`<ul>`) is better.